ingest_test.go

```diff
@@ -159,6 +159,44 @@ func TestExamplesInTheWild(t *testing.T) {
 				assert.EqualError(t, suites[0].Tests[3].Error, "NullPointerException")
 			},
 		},
+		{
+			title:    "phpunit example",
+			filename: "testdata/phpunit.xml",
+			check: func(t *testing.T, suites []Suite) {
```

Comment:
Add an assertion about the updated `Suite.Aggregate()` behavior.

Reply:
found a bug there